src/app.ts

```diff
@@ -100,6 +100,15 @@ async function app() {
     return;
   }
 
+  const fhirR4Dependency = config.dependencies['hl7.fhir.r4.core'];
```

Comment:
I think you may want to make this `config.dependencies?.['hl7.fhir.r4.core']`. That way if someone doesn't specify a dependencies list at all, we get your nice graceful error instead of an unhandled promise rejection.

Reply:
Ah good idea! I'll add that in.